### ISSUE TITLE
Exception for a new user/browser

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#101]
With the new version of Sirepo (from 02/29/2016) when I try to use use it for a new user/browser I cannot see any example and I get the following exception:
```
10.0.2.2 - - [29/Feb/2016 23:26:41] "POST /simulation-list HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/beaker/middleware.py", line 155, in __call__
    return self.wrap_app(environ, session_start_response)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/vagrant/src/mrakitin/sirepo/sirepo/server.py", line 482, in app_simulation_list
    sdb.iterate_simulation_datafiles(simulation_type, sdb.process_simulation_list, search),
  File "/home/vagrant/src/mrakitin/sirepo/sirepo/simulation_db.py", line 93, in iterate_simulation_datafiles
    str(simulation_dir(simulation_type).join('*', SIMULATION_DATA_FILE)),
  File "/home/vagrant/src/mrakitin/sirepo/sirepo/simulation_db.py", line 186, in simulation_dir
    d = _user_dir().join(simulation_type)
  File "/home/vagrant/src/mrakitin/sirepo/sirepo/simulation_db.py", line 307, in _user_dir
    _user_dir_create()
  File "/home/vagrant/src/mrakitin/sirepo/sirepo/simulation_db.py", line 324, in _user_dir_create
    for s in _examples(app_name):
NameError: global name '_examples' is not defined
```
----
Steps to reproduce:
- remove `run` folder from `/home/vagrant/src/radiasoft/sirepo/`;
- try to open the following page - http://localhost:8000/srw#/simulations - in my case the list of calculations/examples is empty.

**Comment by robnagler** [comment]
Fixed b53205b28f75854aea83116f0e90e5ee9b8b98fe

**Comment by mrakitin** [comment]
Thanks Rob! It works fine now.
